biztalk/install-and-config-guides/upgrade-to-biztalk-server-2020.md

```diff
@@ -251,6 +251,9 @@ You cannot roll back to BizTalk Server 2016.
 
 - **Restart applications**: Restart all deployed applications that are upgraded.
 
+> [!IMPORTANT]
+> Applicable starting `BizTalk Server 2020`, `TIBCO Enterprise Message Service Adapter` is supported in 64 bit host only.
+
```

Comment:
Add this as bullet point in Before upgrade. and write specific post upgrade action to update the host running this adapter. You probably also need to consider that there may be other adapters running on the same host and update to 64-bit host may not be straight forward.